runtime/src/main/java/org/corfudb/security/tls/TlsUtils.java

```diff
@@ -151,21 +153,35 @@ public Path getCertStore() {
 
         @AllArgsConstructor
         @Getter
+        @ToString
         class TrustStoreConfig implements CertStoreConfig {
+            public static final Path DEFAULT_DISABLE_CERT_EXPIRY_CHECK_FILE = Paths.get(
+                    "/", "config", "corfu", "DISABLE_CERT_EXPIRY_CHECK"
+            );
+
             /**
-             * KeyStore or TrustStore path
+             * TrustStore path
              */
             private final Path trustStoreFile;
             private final Path passwordFile;
-
-            public static TrustStoreConfig from(String trustStorePath, String passwordFile) {
-                return new TrustStoreConfig(Paths.get(trustStorePath), Paths.get(passwordFile));
+            private final Path disableCertExpiryCheckFile;
+
+            public static TrustStoreConfig from(String trustStorePath, String passwordFile, Path disableCertExpiryCheckFile) {
+                return new TrustStoreConfig(
+                        Paths.get(trustStorePath),
+                        Paths.get(passwordFile),
+                        disableCertExpiryCheckFile
+                );
             }
 
             @Override
             public Path getCertStore() {
                 return trustStoreFile;
             }
+
+            public boolean isCertExpiryCheckEnabled() {
+                return !Files.exists(disableCertExpiryCheckFile);
+            }
```

Comment:
Is there a requirement for this file-based argument passing? In other words, how come the flag `--disable-cert-expiry-check-file` is not enough?